LocalSettings.php

```diff
@@ -1588,11 +1588,31 @@
 					'src' => 'https://static.miraheze.org/lhmnwiki/5/58/Footer.SN.xyz.svg',
 					'url' => 'https://songngu.xyz',
 					'alt' => 'Dự án được bảo quản bởi SongNgư.xyz',
+					"height" => "32",
+				        "width" => "200"
 				],
 				'miraheze' => [
-					'src' => 'https://static.miraheze.org/commonswiki/f/ff/Powered_by_Miraheze.svg',
+					'src' => 'https://static.miraheze.org/lhmnwiki/1/1c/Miraheze.svg',
```

Comment:
is there any reason for changing the "powered by" here?

Reply:
I wanted to give the MH icon the personality of my wiki while still giving the hosting platform the credit it deserved. There should be no infringement at all as it still fully shows that MH have a large part of making my wiki possible.